packages/storefront/README.md

```diff
@@ -62,3 +62,26 @@ var app = new EmberApp({
 });
 ```
 
+## Testing 
+
+```bash
+npm install
+bower install
+# current Ember version
+ember test
+# all supported Ember versions
+npm test
+```
+
+## Contributing 
+
+In the addon folder
+```bash
+npm link
+```
+
+In your project folder
+```bash
+npm link spree-ember-storefront
+ember generate spree-ember-storefront
+```
```

Comment:
This is useful info - however we use the README files for building http://spree-ember.com 

I think this stuff should go in CONTRIBUTING.md (it's in there, but buried)  I also like to discourage people using npm link and instead use bootstrap.rb in the root of the project

Can we pull this out of the READMEs and I'll update CONTRIBUTING ?